### DISCUSSION TITLE
Users module should contains more data

### DISCUSSION THREAD

**Discussion by waran70** [YetiForceCompany/YetiForceCRM#14946] [Feature improvement]
In daily scenario, users is an employee/agent. It would be necessary to indicate in workflows variable (emails templates, pdf) user's position, Title, office address, department etc

**Comment by mariuszkrzaczkowski** [comment]
For this, an employee module was created

**Reply by diderich** [reply]
One one hand, I do like the clear distinction between system users/identity (Users) and firm employees (OSSEmployees). Nevertheless it introduces in some situations redundancy and/or complexity. For example, I have set-up reports/KPI queries that calculate costs of an the activities (OSSTimeControl) of an employee (OSSEmployee) based on their cost to the firm, e.g., salary (a custom field I added). This requires relating an employee to a user (1-1 relationship) to do the calculation and assuming that the record identity is identical to the performing user. As the system is not primarily designed as an ERP (Enterprise Resource Planing) system (although I mis-use it for that in some areas), distinguishing between users and employees is sound to me. I consider it a design choice (similar to the design choice that an e-mail address is not a system identity).

**Reply by mariuszkrzaczkowski** [reply]
We associate the employee with the user using the owner field, there are coded mechanisms that take into account this relationship.

Certainly it requires expansion, what is now just the beginning

**Reply by s2b-git** [reply]
At some point was thinking about following: Cancel users module and instead in modules "Accounts, Contacts, Employees, Vendors" create checkbox "System user" so this particular entity becoming fully legit system user. Obviously this will need massive redesign, but... it would be a method to get away from standard (possibly old fashioned) design. That gonna be sort of like Wordpress, Joomla etc users are managed.
I have some projection of enormous advantages, but I guess there would be also some votes down?

Best
